app/api/api/views.py

```diff
@@ -413,6 +422,52 @@ def perform_destroy(self, instance):
         instance.delete()
 
 
+class StructuralMappingTableAPIView(APIView):
+    def post(self, request, *args, **kwargs):
+        try:
+            body = request.data
+        except ValueError:
+            body = {}
+        if (
```

Comment:
Instead of these parameters for the various ways of downloading the rules - could we just use the [Accept](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Accept) header on the request?

As in - json/csv/svg in the header, rather than `download_rules_as_csv` in the body.